knowing-more-about-ourselves.rst

```diff
@@ -0,0 +1,155 @@
+----------------------------
+
+As you may already know, Read the Docs has a strict `privacy policy <https://docs.readthedocs.io/en/stable/privacy-policy.html>`_
+which we follow to ensure we are protecting the privacy of our users, authors and readers.
+It's still worth mentioning that most of the data we collect is publicly available,
+or is generated by our own service.
+
+The data generated by Read the Docs which we already had access to was mostly high-level about our database objects.
+A few examples would be:
+
+* project configuration (e.g. PR build enabled, default branch)
+* build metadata (e.g. number of builds, length of builds)
+* platform information (redirects, webhooks, search queries, pageviews)
+
+With our new data collections project,
+we were able to collect a lot of new, interesting data.
+This new data allows us to answer a whole new set of questions,
+and is mainly related to the build process
+The new information that we're collecting includes:
+
+* Python dependencies installed during a documentation build
+* Apt packages installed prior to building docs
+* The entire contents of the `Read the Docs configuration file <https://docs.readthedocs.io/en/stable/config-file/index.html>`_
+
+This allows us to answer questions like:
+
+- "How many projects are using Sphinx v4.5.0 in their builds?"
+- "What are the most used Sphinx extensions?"
+- "How fast is the adoption of a new Sphinx version once it is released?"
+
+Plotting it nicely
+------------------
+
+The `EthicalAds <https://www.ethicalads.io/>`_ team, who is building our privacy-first ad network,
+suggested using `Metabase <https://www.metabase.com/>`_ since they were already using it for a similar use case and they were happy with it.
+We connected our PostgreSQL replica database to it and started plotting this data to get some insights.
+We found awesome things we want to start sharing with all the documentation community,
+since we think it will useful not only for ourselves but also for package maintainers.
+
+Sphinx extensions used in the last 15 days
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. image:: /img/sphinx-extensions.png
+   :align: center
+   :width: 100%
+
+.. note::
+
+   We excluded ``sphinx-rtd-theme`` and ``sphinxcontrib-websupport`` because they are too high and break the relation with the others in the graph.
+   Also, only extensions used by more than 10 projects are showed.
+
+Sphinx adoption over time
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. image:: /img/sphinx-adoption.png
+   :align: center
+   :width: 100%
+
+
+Projects using MyST to write Markdown in Sphinx
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. image:: /img/myst-parser.png
+   :align: center
+   :width: 100%
+
+
+VCS types used by projects with successful builds in last year
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. image:: /img/project-vcs-types.png
+   :align: center
+   :width: 100%
+
+
+Docker image used in the last 15 days
```

Comment:
What is `default` here? Also, is `ubuntu-latest` a real option, or an error?

Reply:
`default` means the user did not specify anything in their config file. IIRC, on config file v2, this is `latest`.

`ubuntu-latest` is an invalid image and the build should have failed. I'm not sure why it's listed here. I didn't notice it when taking the screenshot. That's wrong and it shouldn't be there. I will take a look at the query and fix it. Thanks!

Reply:
Hrm... I checked the query. The user used `build.os: ubuntu-latest`, and the config was translated finally to `build.image: readthedocs/build:latest`. That's why it succeed. However, this is an error and we should raise a build error or similar.

I will remove it from this query for now because I don't want to communicate `ubuntu-latest` on the blog post since it's an invalid option.

Reply:
This is done. I updated the plot with more accurate data.